src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 101;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int randomInt = random.nextInt(FIGURE_COUNT);
+        Ball ball = new Ball();
+        ball.setNumber(randomInt);
+        ball.setColor(colorSupplier.getRandomColor());
+        return ball;
```

Comment:
would be better if all-args constuctor is introduced and used
```suggestion
        return new Ball(randomInt, colorSupplier.getRandomColor());
```